src/Famix-Traits/FamixTNamed.trait.st

```diff
@@ -32,7 +32,7 @@ FamixTNamed classSide >> named: aString [
 { #category : #testing }
 FamixTNamed >> hasUniqueMooseNameInModel [
 	" all subclasses should call the method: 'resetMooseName' if a change causes the moosename to change "
-	^true
+	^false
```

Comment:
I do not agree with this change.

Named entity should always have unique name.